### DISCUSSION TITLE
Lack of specified sprite path in some vanilla items makes it impossible to make variants of these items

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#14806] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

for example combat diving suit its Wearable texture is "combat_DivingSuit.png"
It wont cause any issue in vanilla
but if a mod want to make a variant of combat diving suit which only modify its helmet sprite the game cant find its sprites

### Reproduction steps

1. make variants of items that have no specified path

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by ubertpendragon** [comment]
"Some"

I wish that were true

**Reply by Regalis11** [reply]
Could you elaborate? 🤔

**Reply by ubertpendragon** [reply]
There's just a lot of them, gives you a nasty surprise when you copy-paste items for overrides and you get a whole lot of erros 😅

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/dde6bda567e32ca3430ff66ed495a833ad911116 (commit in our private repo).

Instead of switching to full paths, I made the game look for the files in the folder the base item is in. I suppose that could potentially cause issues in some mods that have worked around the problem by placing sprites in the mod folder too, but I feel like it's the "correct" way to solve this: the game should assume the item is supposed to use the sprites of the base item if they have not been overridden/modified.

**Comment by NilanthAnimosus** [comment]
Closing as fixed and merged. Should be fixed now in a future update. likely past the 2nd unto the breach hotfix.
